src/layout/header/Header.tsx

```diff
@@ -85,56 +85,57 @@ export const Header = () => {
         aria-label={t('common.main')}
         component="nav"
         sx={{
-          display: 'grid',
-          justifyItems: 'center',
-          gridTemplateAreas: '"logo search new-result user-menu"',
-          gridTemplateColumns: { xs: 'auto auto 1fr auto', md: 'auto 1fr 10fr auto' },
+          display: 'flex',
+          flexWrap: 'wrap',
+          justifyContent: 'space-between',
           gap: { xs: '0.5rem', sm: '1rem' },
           px: '1rem',
         }}>
-        <Logo />
-        <MenuIconButton
-          color="inherit"
-          sx={{ gridArea: 'search' }}
-          title={t('common.search')}
-          isSelected={location.pathname === UrlPathTemplate.Root || currentPath === UrlPathTemplate.Root}
-          to={UrlPathTemplate.Root}>
-          <SearchIcon fontSize="large" />
-        </MenuIconButton>
+        <Box sx={{ display: 'flex', gap: '2rem' }} flex={1}>
```

Comment:
Litt forvirrende at to flex-attributter er i sx og en er som en prop? Foreslår å enten sette alt i sx eller som props (ikke testet, men regner med dette blir det samme)
```suggestion
        <Box sx={{ display: 'flex', gap: '2rem', flex: 1 }}>
```